packages/gtmcore/gtmcore/inventory/repository.py

```diff
@@ -267,7 +267,12 @@ def checkout_id(self) -> str:
                     self._checkout_id = cf.read()
             else:
                 # Create a new checkout ID and file
-                self._checkout_id = f"{self.key}|{self.git.get_current_branch_name()}|{uuid.uuid4().hex[0:10]}"
+                try:
+                    self._checkout_id = f"{self.key}|{self.git.get_current_branch_name()}|{uuid.uuid4().hex[0:10]}"
```

Comment:
We already have a wrapped version. This is another example of how we're not using a clear architecture for git (which seems important, since git is complicated)

```suggestion
                    self._checkout_id = f"{self.key}|{self.active_branch}|{uuid.uuid4().hex[0:10]}"
```

Reply:
Good catch

Reply:
yeah...in general a _ton_ of git stuff has been moved to subprocess calls from the git class. I'm not thrilled with it, but it works for now. We'll need to consolidate things relatively soon back into the git class, probably re-implementing everything we need via subprocess calls and cleaning out the rest, including GitPython.